x-pack/plugin/build.gradle

```diff
@@ -8,16 +8,8 @@ import java.nio.file.Path
 import java.nio.file.StandardCopyOption
 import org.elasticsearch.gradle.test.RunTask;
 
-apply plugin: 'elasticsearch.es-meta-plugin'
```

Comment:
I talked with @rjernst about this part - now that xpack is bundled as part of the default distribution the run task was actually the last part that needed this.

Reply:
Sorry for the confusion. I only meant to move the run task configuration in this PR. This removal was accidental, and from another branch.